airgun/views/activationkey.py

```diff
@@ -14,13 +19,28 @@ def is_displayed(self):
 
 
 class ActivationKeyDetailsView(BaseLoggedInView):
+
     name = TextInput(id='name')
     description = TextInput(id='description')
     unlimited_hosts = Checkbox(name='limit')
     max_hosts = TextInput(id='max_hosts')
     submit = Text("//button[contains(@ng-click, 'handleSave')]")
+    lce = CheckboxGroup()
+    content_view = Select(id='content_view_id')
 
     @property
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.name, exception=False) is not None
+
+
+class ActivationKeyEditView(BaseLoggedInView):
+    return_to_all = Text("//a[text()='Activation Keys']")
+    action_list = SelectActionList()
+    dialog = ConfirmationDialog()
```

Comment:
Have you checked cfme code? Separate dialog window as a widget doesn't sound right to me, i'd rather expect it to be a view, which appears during this view's `fill`. Does it make sense?

Reply:
Yes, I am sure. It is simple control with two buttons